cmd/topicctl/subcmd/rebalance.go

```diff
@@ -0,0 +1,318 @@
+package subcmd
+
+import (
+	"context"
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/segmentio/topicctl/pkg/admin"
+	"github.com/segmentio/topicctl/pkg/apply"
+	"github.com/segmentio/topicctl/pkg/cli"
+	"github.com/segmentio/topicctl/pkg/util"
+	"github.com/segmentio/topicctl/pkg/config"
+	log "github.com/sirupsen/logrus"
+)
+
+var rebalanceCmd = &cobra.Command{
+	Use:     "rebalance",
+	Short:   "rebalance all topic configs for a kafka cluster",
+	PreRunE: rebalancePreRun,
+	RunE:    rebalanceRun,
+}
+
+type rebalanceCmdConfig struct {
+	brokersToRemove              []int
+	brokerThrottleMBsOverride    int
+	dryRun                       bool
+	partitionBatchSizeOverride   int
+	pathPrefix                   string
+	autoContinueRebalance        bool
```

Comment:
Is there a use case for needing this flag in the context of a "rebalance everything" command? Seems like it should be the default behavior.

Reply:
Will be resolved in next PR